.github/workflows/push-mimir-build-image.yml

```diff
@@ -70,17 +70,15 @@ jobs:
         id: notification
         run: | 
           if [ ${{ steps.check_build.outputs.build }} == 'true' ]; then
-           gh pr comment $PR_URL --body "**Building new version of `mimir-build-image`**. After image is built and pushed to Docker Hub, \
-           a new commit will automatically be added to this PR with new image version `$IMAGE:$TAG`. This can take up to 1 hour."
+           gh pr comment $PR_URL --body "**Building new version of mimir-build-image**. After image is built and pushed to Docker Hub, \
+           a new commit will automatically be added to this PR with new image version ${{ steps.prepare.outputs.image }}: ${{ steps.compute_hash.outputs.tag }}. This can take up to 1 hour."
           else
             echo "This PR will not trigger a build of mimir-build-image"
-            gh pr comment $PR_URL --body "**Not building new version of `mimir-build image`**. This PR modifies `mimir-build-image/Dockerfile`, but the image `$IMAGE:$TAG` already exists."
+            gh pr comment $PR_URL --body "**Not building new version of mimir-build-image**. This PR modifies mimir-build-image/Dockerfile, but the image ${{ steps.prepare.outputs.image }}: ${{ steps.compute_hash.outputs.tag }} already exists."
           fi
         env:
           PR_NUMBER: ${{ github.event.pull_request.number }}
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
-          TAG: ${{ steps.compute_hash.outputs.tag }}
-          IMAGE: ${{ steps.prepare.outputs.image }}
 
```

Comment:
I don't think this is necessary: the issue is that the backticks in the comment description are being interpreted by Bash as inline commands (same as including `$(...)` in a double-quoted string). In the [logs from a recent run](https://github.com/grafana/mimir/actions/runs/6107538001/job/16574744536#step:10:17), Bash is complaining about trying to run what would be between the backticks here.

Instead, I think we need to escape the backticks. https://stackoverflow.com/a/2310284 suggests replacing `` ` `` with `` \\\` `` in the strings.

Reply:
Thanks done, I just removed the backflash on mimir-build-image and escaped the one for image and tag as your comments.